.github/workflows/1_2_b_bump_extension_only.yml

```diff
@@ -29,12 +29,12 @@ jobs:
         with:
           node-version: '20'
           cache: 'npm'
-      - name: Install Dependencies
-        run: npm install && npm run bootstrap
-        shell: bash
```

Comment:
ℹ️ Installing dependencies and then switching branch blew up when `yarn.lock` had small differences due to different node/npm/runtime environments. Making the following `git checkout` fail. 
In general we should first checkout what we want, then install dependencies.